scripts/main.py

```diff
@@ -686,7 +692,11 @@ def on_ui_tabs():
                         gr.HTML('Remove <b>selected</b> tags from the images displayed.')
                         btn_remove_selected = gr.Button(value='Remove selected tags', variant='primary')
                         tag_select_ui_remove.create_ui(get_filters, cfg_batch_edit.sory_by, cfg_batch_edit.sort_order, cfg_batch_edit.sw_prefix, cfg_batch_edit.sw_suffix, cfg_batch_edit.sw_regex)
-                    
+                with gr.Tab(label='Extras'):
```

Comment:
I wasn't sure what to name this new tab, so I took inspiration from the main webui's "Extras" tab